compound_disease/compound_treats_disease/model_calibration_experiment/model_calibration_experiment.py

```diff
@@ -0,0 +1,102 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
```

Comment:
I'm not sure what this nb is doing.  What are you calibrating?

Reply:
This part comes in after training the discriminative model. I'm using a deep learning model and deep learning models aren't calibrated correctly ([paper here](https://arxiv.org/pdf/1706.04599.pdf)). Based on this fact, I'm using a calibrator to make my predictions more trustworthy.